Add linq using, erase SDK requirement

kendrahavens opened the pull request:
- You need a System.Linq using statement to compile this. 
- You do not need to clone the GitHub repo to run this sample so it shouldn't be under prerequisites. A link to the WPF sample can be in a further reading section at the bottom.

azurecla commented:
Hi __@alexwun__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alexwun). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@alexwun - Future freshness updates would be best made in the private repo, please. Thanks

kendrahavens committed:
Add linq using, erase SDK requirement

- You need a System.Linq using statement to compile this. 
- You do not need to clone the GitHub repo to run this sample so it shouldn't be under prerequisites. A link to the WPF sample can be in a further reading section at the bottom.
